hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/checksum/ContainerChecksumTreeManager.java

```diff
@@ -316,12 +317,12 @@ public static boolean hasContainerChecksumFile(ContainerData data) {
    */
   @VisibleForTesting
   public static File getContainerChecksumFile(ContainerData data) {
```

Comment:
If we are using this when building the tar we should remove the `@VisibleForTesting` annotation.